moj-fe/app/Http/Controllers/HealthController.php

```diff
@@ -3,17 +3,88 @@
 namespace App\Http\Controllers;
 
 use Symfony\Component\HttpFoundation\Response;
+use Illuminate\Support\Facades\Log;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Psr7;
+use GuzzleHttp\Client;
 
 class HealthController extends Controller
 {
-    /**
-     * @return \Symfony\Component\HttpFoundation\Response
-     */
-    function checkHealth()
+
+    protected $client;
+
+    protected $response;
+
+    protected $cms;
+
+    protected $cmsHealth;
+
+    public function getResponse()
+    {
+        return $this->response;
+    }
+
+    public function getCmsHealth()
+    {
+        return $this->cmsHealth;
+    }
+
+    public function __construct()
+    {
+        $this->response = new Response();
+        $this->client = new Client(
+          [
+            'base_uri' => config('app.api_uri'),
+            'timeout' => config('app.timeout'),
+          ]
+        );
+    }
+
+    public function frontEndCheckHealthEndpoint()
+    {
+        $this->setHttpResponse(200);
```

Comment:
if we want separate checks we can have `/frontend-health` and `/backend-health`.

But the main `/health` check should be a full-stack check of all the bits.

Example from nomis web https://notm-dev.hmpps.dsd.io/health
```
{
  "name": "elite2-web",
  "version": "2018-03-05.2261",
  "description": "Elite 2 Web",
  "api": {
    "status": "UP",
    "healthInfo": {
      "status": "UP",
      "version": "1.0.21.2224"
    },
    "diskSpace": {
      "status": "UP",
      "total": 10725883904,
      "free": 10512019456,
      "threshold": 10485760
    },
    "db": {
      "status": "UP",
      "database": "Oracle",
      "hello": "Hello"
    }
  }
}
```

Reply:
I've switched to a full stack endpoint and changed the JSON to use a similar format to the example you gave.